src/com/company/Entities/Hall.java

```diff
@@ -0,0 +1,72 @@
+package com.company.Entities;
+
+public class Hall {
+    private String name; //identifier
+    private int floor;
+    private int seatsNumber;
+    private boolean available = true;
+    private int rows, columns; //for the seats matrix

```

Comment:
we usually declare fields sepparately even if they are of same type as best practice